src/System.Xml.ReaderWriter/tests/XmlWriter/System.Xml.RW.XmlWriter.Tests.csproj

```diff
@@ -29,6 +29,10 @@
       <Project>{be8ed8c1-c314-4c4e-a929-64c9c8b3552a}</Project>
       <Name>XunitTraitsDiscoverers</Name>
     </ProjectReference>
+    <ProjectReference Include="..\..\..\System.Text.Encoding.CodePages\src\System.Text.Encoding.CodePages.csproj">
```

Comment:
I know we've had this conversation before, but I still don't understand why our test projects pull in from a NuGet package when the corresponding, potentially more recent source is available for P2P reference in the repo.  Using a P2P reference seems like a great way to get more validation of changes, essentially for free.  Moreover, the usage from such dependent test libraries is likely to be more closely aligned with real-world usage than the tests for that specific library, because this test library is using the project for its intended purpose rather than explicitly trying to exercise it.